### ISSUE TITLE
Create SynqStreamer in Swift and add to SYNQ-Swift

### ISSUE THREAD

**Issue by kjartanvest** [SYNQfm/SYNQ-Swift#3]

**Comment by kjartanvest** [comment]
Using static lib in Swift:
https://github.com/aliceatlas/buildstatic

**Comment by kjartanvest** [comment]
Modules for static libraries: http://blog.bjhomer.com/2015/05/defining-modules-for-custom-libraries.html

**Comment by kjartanvest** [comment]
Build universal binary Swift framework using CocoaPods: https://eladnava.com/publish-a-universal-binary-ios-framework-in-swift-using-cocoapods/

**Comment by batmany13** [comment]
@kjartanvest Question, is this for supporting live streaming?  Could this be integrated in the same SYNQ-Swift repo we already have, instead of having to create two separate ones?

**Comment by kjartanvest** [comment]
Correct, @batmany13, this is for adding live streaming. This will be integrated into SYNQ-Swift (it might be necessary to have a subproject/submodule to generate the static streamer lib, that´s what I did in the Objective-C version. If so, a separate repo must be created for this)
